src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorArrayAccessTest.php

```diff
@@ -82,4 +82,28 @@ public function testIsWritable($collection, $path)
     {
         $this->assertTrue($this->propertyAccessor->isWritable($collection, $path));
     }
+
+    public function getValidPropertyPathsForGetValues()
```

Comment:
The test does not seem to cover enough cases. You at least need to add values that should not be returned. Also add values that will cause exceptions for example.